<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>681</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(35, 37, 39);</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 15pt Helvetica; 
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(70, 70, 82); 
padding: 3px;font:15px;} 
QPushButton:hover{
background: rgb(71, 71, 88);
} 
QPushButton:pressed{
border: 1px solid rgb(150, 149, 178);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 15pt Helvetica; 
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(70, 70, 82); 
padding: 3px;font:15px;} 
QPushButton:hover{
background: rgb(71, 71, 88);
} 
QPushButton:pressed{
border: 1px solid rgb(150, 149, 178);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 15pt Helvetica; 
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(70, 70, 82); 
padding: 3px;font:15px;} 
QPushButton:hover{
background: rgb(71, 71, 88);
} 
QPushButton:pressed{
border: 1px solid rgb(150, 149, 178);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>215</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 15pt Helvetica; 
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(70, 70, 82); 
padding: 3px;font:15px;} 
QPushButton:hover{
background: rgb(71, 71, 88);
} 
QPushButton:pressed{
border: 1px solid rgb(150, 149, 178);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>215</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 15pt Helvetica; 
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(70, 70, 82); 
padding: 3px;font:15px;} 
QPushButton:hover{
background: rgb(71, 71, 88);
} 
QPushButton:pressed{
border: 1px solid rgb(150, 149, 178);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>215</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 15pt Helvetica; 
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(70, 70, 82); 
padding: 3px;font:15px;} 
QPushButton:hover{
background: rgb(71, 71, 88);
} 
QPushButton:pressed{
border: 1px solid rgb(150, 149, 178);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>260</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 15pt Helvetica; 
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(70, 70, 82); 
padding: 3px;font:15px;} 
QPushButton:hover{
background: rgb(71, 71, 88);
} 
QPushButton:pressed{
border: 1px solid rgb(150, 149, 178);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 15pt Helvetica; 
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(70, 70, 82); 
padding: 3px;font:15px;} 
QPushButton:hover{
background: rgb(71, 71, 88);
} 
QPushButton:pressed{
border: 1px solid rgb(150, 149, 178);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 15pt Helvetica; 
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(70, 70, 82); 
padding: 3px;font:15px;} 
QPushButton:hover{
background: rgb(71, 71, 88);
} 
QPushButton:pressed{
border: 1px solid rgb(150, 149, 178);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>305</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 15pt Helvetica; 
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(70, 70, 82); 
padding: 3px;font:15px;} 
QPushButton:hover{
background: rgb(71, 71, 88);
} 
QPushButton:pressed{
border: 1px solid rgb(150, 149, 178);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>305</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 15pt Helvetica; 
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(70, 70, 82); 
padding: 3px;font:15px;} 
QPushButton:hover{
background: rgb(71, 71, 88);
} 
QPushButton:pressed{
border: 1px solid rgb(150, 149, 178);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>305</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 20pt Helvetica; 
color:  rgb(30, 31, 34);
border-radius:3px;
background: rgb(191, 156, 204); 
padding: 5px;
} 
      QPushButton:hover{
      background: rgb(202, 165, 215);
      } 
      QPushButton:pressed{
      border: 1px solid rgb(255,255,225);
      }</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>215</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt Helvetica;
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(118, 114, 122); 
padding: 5px;font: 15px;
} 
QPushButton:hover{
background: rgb(130, 124, 135);
} 
 QPushButton:pressed{
border: 1px solid rgb(195,195,195);
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>260</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt Helvetica;
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(118, 114, 122); 
padding: 5px;font: 15px;
} 
QPushButton:hover{
background: rgb(130, 124, 135);
} 
 QPushButton:pressed{
border: 1px solid rgb(195,195,195);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>170</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt Helvetica;
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(118, 114, 122); 
padding: 5px;font: 15px;
} 
QPushButton:hover{
background: rgb(130, 124, 135);
} 
 QPushButton:pressed{
border: 1px solid rgb(195,195,195);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>305</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt Helvetica;
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(118, 114, 122); 
padding: 5px;font: 15px;
} 
QPushButton:hover{
background: rgb(130, 124, 135);
} 
 QPushButton:pressed{
border: 1px solid rgb(195,195,195);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_17">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>125</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt Helvetica;
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(118, 114, 122); 
padding: 5px;font: 15px;
} 
QPushButton:hover{
background: rgb(130, 124, 135);
} 
 QPushButton:pressed{
border: 1px solid rgb(195,195,195);
}</string>
    </property>
    <property name="text">
     <string>⌫</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_18">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>125</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt Helvetica;
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(118, 114, 122); 
padding: 5px;font: 15px;
} 
QPushButton:hover{
background: rgb(130, 124, 135);
} 
 QPushButton:pressed{
border: 1px solid rgb(195,195,195);
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_19">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>125</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt Helvetica;
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(118, 114, 122); 
padding: 5px;font: 15px;
} 
QPushButton:hover{
background: rgb(130, 124, 135);
} 
 QPushButton:pressed{
border: 1px solid rgb(195,195,195);
}</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_20">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>125</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt Helvetica;
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(118, 114, 122); 
padding: 5px;font: 15px;
} 
QPushButton:hover{
background: rgb(130, 124, 135);
} 
 QPushButton:pressed{
border: 1px solid rgb(195,195,195);
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_21">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>305</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt Helvetica;
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(118, 114, 122); 
padding: 5px;font: 15px;
} 
QPushButton:hover{
background: rgb(130, 124, 135);
} 
 QPushButton:pressed{
border: 1px solid rgb(195,195,195);
}</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_22">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt Helvetica;
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(118, 114, 122); 
padding: 5px;font: 15px;
} 
QPushButton:hover{
background: rgb(130, 124, 135);
} 
 QPushButton:pressed{
border: 1px solid rgb(195,195,195);
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_23">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>125</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt Helvetica;
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(118, 114, 122); 
padding: 5px;font: 15px;
} 
QPushButton:hover{
background: rgb(130, 124, 135);
} 
 QPushButton:pressed{
border: 1px solid rgb(195,195,195);
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_24">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt Helvetica;
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(118, 114, 122); 
padding: 5px;font: 15px;
} 
QPushButton:hover{
background: rgb(130, 124, 135);
} 
 QPushButton:pressed{
border: 1px solid rgb(195,195,195);
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_25">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>215</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt Helvetica;
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(118, 114, 122); 
padding: 5px;font: 15px;
} 
QPushButton:hover{
background: rgb(130, 124, 135);
} 
 QPushButton:pressed{
border: 1px solid rgb(195,195,195);
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_26">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt Helvetica;
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(118, 114, 122); 
padding: 5px;font: 15px;
} 
QPushButton:hover{
background: rgb(130, 124, 135);
} 
 QPushButton:pressed{
border: 1px solid rgb(195,195,195);
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_27">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt Helvetica;
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(118, 114, 122); 
padding: 5px;font: 15px;
} 
QPushButton:hover{
background: rgb(130, 124, 135);
} 
 QPushButton:pressed{
border: 1px solid rgb(195,195,195);
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_28">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>305</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt Helvetica;
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(118, 114, 122); 
padding: 5px;font: 15px;
} 
QPushButton:hover{
background: rgb(130, 124, 135);
} 
 QPushButton:pressed{
border: 1px solid rgb(195,195,195);
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_29">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>215</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt Helvetica;
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(118, 114, 122); 
padding: 5px;font: 15px;
} 
QPushButton:hover{
background: rgb(130, 124, 135);
} 
 QPushButton:pressed{
border: 1px solid rgb(195,195,195);
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_30">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>125</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt Helvetica;
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(118, 114, 122); 
padding: 5px;font: 15px;
} 
QPushButton:hover{
background: rgb(130, 124, 135);
} 
 QPushButton:pressed{
border: 1px solid rgb(195,195,195);
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>35</y>
      <width>691</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
      color: rgb(255, 255, 255); 
      border: 0.5px solid rgb(90, 88, 91);
      border-radius: 5px; 
      padding: 3px; 
      font: 30pt Helvetica;
      background: rgb(35, 37, 39);
      }; </string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_31">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>125</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt Helvetica;
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(118, 114, 122); 
padding: 5px;font: 15px;
} 
QPushButton:hover{
background: rgb(130, 124, 135);
} 
 QPushButton:pressed{
border: 1px solid rgb(195,195,195);
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_32">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>170</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt Helvetica;
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(118, 114, 122); 
padding: 5px;font: 15px;
} 
QPushButton:hover{
background: rgb(130, 124, 135);
} 
 QPushButton:pressed{
border: 1px solid rgb(195,195,195);
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
font: 15pt Helvetica;
color: rgb(74, 73, 88);
border-radius: 1px;
background: rgb(214, 211, 217); 
padding: 5px;font: 15px;
} </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_34">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>350</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt Helvetica;
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(118, 114, 122); 
padding: 5px;font: 15px;
} 
QPushButton:hover{
background: rgb(130, 124, 135);
} 
 QPushButton:pressed{
border: 1px solid rgb(195,195,195);
}</string>
    </property>
    <property name="text">
     <string>clear x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_33">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt Helvetica;
color: rgb(255, 255, 255);
border-radius: 3px;
background: rgb(118, 114, 122); 
padding: 5px;font: 15px;
} 
QPushButton:hover{
background: rgb(130, 124, 135);
} 
 QPushButton:pressed{
border: 1px solid rgb(195,195,195);
}</string>
    </property>
    <property name="text">
     <string>graph</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
